src/Components/Bookshelf/Bookshelf.tsx

```diff
@@ -81,7 +81,8 @@ class Bookshelf extends React.Component<BookshelfProps, BookshelfState> {
     } else {
       return (
         <div className='bookshelf-background'>
-          {this.props.queryID && <h2 className='bookType'>{formatBookshelfTitle(this.props.queryID.split('-').join(' ')) || this.props.favoritesHeader}</h2>}
+          {this.props.queryID && <h2 className='bookType'>{formatBookshelfTitle(this.props.queryID.split('-').join(' '))}</h2>}
+          {!this.props.queryID && <h2 className='bookType'>{this.props.favoritesHeader}</h2>}
           <section className='bookshelf'>
             {bookCards}
           </section>
```

Comment:
Great job getting this to work in a different condition that says queryID is falsy.